templates/projects/tasks_browse.webpack.ejs

```diff
@@ -644,15 +644,17 @@
 </div>
 
 <!-- Project owner, subadmin that are project co-owners and admins can edit/delete task properties -->
-{% if (current_user.subadmin and current_user.id in project.owners_ids) or current_user.admin %}
+{% if current_user.id in project.owners_ids or current_user.admin %}
 <div id="context-menu" class="dropdown clearfix">
     <ul class="dropdown-menu" role="menu" aria-labelledby="dropdown-menu" style="display:block;position:absolute;">
+        {% if (current_user.subadmin and current_user.id in project.owners_ids) or current_user.admin %}
```

Comment:
It looks like this condition could be simply rewritten as:

```
{% if current_user.subadmin or current_user.admin %}
```

Reply:
Yes, I was using the previously written condition on line 647 to apply to the priority and redundancy edit. 
If you think that nested if statements are more readable here, then I can change it! Let me know.

Reply:
If the logic is duplicated, then we could optimize per above, to reduce code error.

*I wasn't sure if the logic is absolutely necessary here or if it is indeed a duplicate of the parent condition.*

Reply:
okay, I will change it according to your recommendation